src/app/(privateProfile)/calculator/page.tsx

```diff
@@ -1,9 +1,433 @@
+
+				<div className={style.currencySelectionWrapper}>
+					<p className={style.currencySelectionWrapper__description}>Рассчитать в:</p>
+					<div className={style.currencySelectionRadioGroup}>
+						<input
+							name="currency"
+							type="radio"
+							id="currencySelectionRadioGroup__usd"
+							className={style.currencySelectionRadioGroup__input}
+						/>
+						<label htmlFor="currencySelectionRadioGroup__usd" className={style.currencySelectionRadioGroup__label}>
+							<p className={style.currencySelectionRadioGroup__text}>$</p>
+						</label>
+						<input
+							name="currency"
+							type="radio"
+							id="currencySelectionRadioGroup__rub"
+							className={style.currencySelectionRadioGroup__input}
+							defaultChecked
+						/>
+						<label htmlFor="currencySelectionRadioGroup__rub" className={style.currencySelectionRadioGroup__label}>
+							<p className={style.currencySelectionRadioGroup__text}>₽</p>
+						</label>
+						<input
+							name="currency"
+							type="radio"
+							id="currencySelectionRadioGroup__eur"
+							className={style.currencySelectionRadioGroup__input}
+						/>
+						<label htmlFor="currencySelectionRadioGroup__eur" className={style.currencySelectionRadioGroup__label}>
+							<p className={style.currencySelectionRadioGroup__text}>€</p>
+						</label>
+					</div>
+				</div>
+
+				<div className={style.calculationWrapper}>
+					<form className={style.calculatorFormContentWrapper}>
+						<div className={style.selectionOfCalculationContainer}>
+							<input
+								name="select-of-calculation"
+								type="radio"
+								id="selectionOfCalculationContainer__real"
+								className={style.selectionOfCalculationContainer__input}
+								defaultChecked
+							/>
+							<label
+								htmlFor="selectionOfCalculationContainer__real"
+								className={style.selectionOfCalculationContainer__label}>
+								<p className={style.selectionOfCalculationContainer__text}>Недвижимость</p>
+							</label>
+							<input
+								name="select-of-calculation"
+								type="radio"
+								id="selectionOfCalculationContainer__credit"
+								className={style.selectionOfCalculationContainer__input}
+							/>
+							<label
+								htmlFor="selectionOfCalculationContainer__credit"
+								className={style.selectionOfCalculationContainer__label}>
+								<p className={style.selectionOfCalculationContainer__text}>Кредит</p>
+							</label>
+						</div>
+
+						<div className={style.propertyValueContainer}>
+							<label className={style.propertyValueContainer__label} htmlFor="propertyValue">
+								Стоимость недвижимости, ₽
+							</label>
+							<div className={style.inputRangeFieldContainer}>
+								<input
+									type="text"
+									value={formatNumber(value)}
+									onChange={handleInputChange}
+									className={style.inputRangeFieldContainer__input}
+									min="0"
+									max="10000000"
+									id="propertyValue"
+								/>
```

Comment:
i think you can use here type number with hiding arrows using css here and in other simimar cases